joomla/joomla-cms
# [5.2] [Guided tours] Add a What's New tour when updating to Joomla 5.2

### Summary of Changes

This PR adds a tour that highlights new features introduced in Joomla 5.2.

### Testing Instructions

Download the update package, not the full install as the tour is only available on updates!

It is possible to find the tour in the list of tours at the top of the Joomla dashboard and run it manually.
Make sure you see images when running the tour (on the first step only) and when editing the tour (System -> Manage -> Guided tours).

### Actual result BEFORE applying this Pull Request

No tour highlighting new features for Joomla 5.2.

### Expected result AFTER applying this Pull Request

What’s New in Joomla 5.2! tour is available.

![image](https://github.com/user-attachments/assets/87835393-40dc-4f8c-bf62-9d74b349186a)

![image](https://github.com/user-attachments/assets/eb6a415f-1300-46c4-9ecf-98e9af097a9c)

![image](https://github.com/user-attachments/assets/1ef4f531-2f44-406b-a635-e8fc00825067)

### Link to documentations
Please select:
